migration-to-PERCS-v2.0.md

```diff
@@ -0,0 +1,222 @@
+for a custom type. Enums are new to PHP since 8.1 and so could not be covered
+by PSR12.
+
+PER-CS addresses aspects on how standardise on coding Enums in new section 9.
+As Enums do not support inheritance non-public methods MUST use the private
+keyword.
+
+When using backed enums, there MUST NOT be a space between the enum name and 
+colon, and there must be exactly one space between the colon and the backing
+type, e.g. "enum Suit: string" or "enum BanExpiry: int".
+
+Enum case declarations MUST use PascalCase capitalization. Enum case
+declarations MUST be on their own line.
+
+An example of this can be seen in the online php documentation at https://www.php.net/manual/en/language.enumerations.backed.php
+
+    <?php
+    enum Suit: string
+    {
+        case Hearts = 'H';
+        case Diamonds = 'D';
+        case Clubs = 'C';
+        case Spades = 'S';
+    }
+    ?>
+
+Constants in Enums MAY be either PascalCase or UPPER_CASE, PascalCase
+is RECOMMENDED, so that it is consistent with case declarations.
+
+
+    <?php
+    enum Size
+    {
+        case Small;
+        case Medium;
+        case Large;
+    
+        public const Huge = self::Large;
+    }
+    ?>
+
+## [Section 10](https://www.php-fig.org/per/coding-style/#10-heredoc-and-nowdoc)
+
+This is a new section about HereDocs and NowDocs.
+
+NowDoc SHOULD be used whereever possible. Heredoc MAY be used where a
+nowdoc does not satisfy.
+
+Declared heredocs or nowdocs MUST begin on the same line as the context
+the declaration is being used in. Subsequent lines in the heredoc
+or nowdoc MUST be indented once past the scope indentation they are
+declared in.
+
+The heredoc MUST be declared on the same line as the variable declaration it's being set against.
+The heredoc MUST be indented once past the indentation of the scope it's declared in.
+
+## [Section 11](https://www.php-fig.org/per/coding-style/#11-arrays)
+
+This too is a new section. It is about arrays.
+
+Arrays MUST be declared using the short array syntax.
+Arrays MUST follow the trailing comma guidelines.
+
+Array declarations MAY be split across multiple lines, where each
+subsequent line is indented once. When doing so, the first value in the
+array MUST be on the next line, and there MUST be only one value per line.
+
+When the array declaration is split across multiple lines, the opening
+bracket MUST be placed on the same line as the equals sign. 
+
+The closing bracket MUST be placed on the next line after the last value. 
+
+There MUST NOT be more than one value assignment per line. 
+
+Value assignments MAY use a single line or multiple lines.
+
+## [Section 12](https://www.php-fig.org/per/coding-style/#12-attributes)
+
+This is a new section about attributes. Attributes are new in PHP 8 and
+offer the ability to add structured, machine-readable metadata information on 
+declarations in code (https://www.php.net/manual/en/language.attributes.overview.php) 
```

Comment:
Again, there's no point in calling out that they're new.  I'm undecided about manual links.

Reply:
This whole section feels like it's just duplicating the actual spec document.  That is prone to confusion if the wording is slightly different in each case.

Reply:
I reduced the introductory paragraph for this section, removing the exposition and the link to the manual.
People can do an internet search...